website/docs/docs/build/packages.md

```diff
@@ -165,27 +165,45 @@ dbt Cloud supports private packages from [supported](#prerequisites) Git repos l
 
 #### Prerequisites
 
-To use native private packages, you must have one of the following Git providers configured in the **Integrations** section of your **Account settings**:
-- [GitHub](/docs/cloud/git/connect-github)
-- [Azure DevOps](/docs/cloud/git/connect-azure-devops)
-- Support for GitLab is coming soon.
-
+- To use native private packages, you must have one of the following Git providers configured in the **Integrations** section of your **Account settings**:
+  - [GitHub](/docs/cloud/git/connect-github)
+  - [Azure DevOps](/docs/cloud/git/connect-azure-devops)
+    - For Azure DevOps, use the `org/repo` path (not the `org_name/project_name/repo_name` path) with the project tier inherited from the integrated source repository.
```

Comment:
I think the important thing to emphasize as a pre-req is that they only work in a single project within ADO. The specific syntax, I think you do a good job of covering below.